src/main/kotlin/no/nav/syfo/syfosyketilfelle/client/SyfosyketilfelleClient.kt

```diff
@@ -0,0 +1,70 @@
+package no.nav.syfo.syfosyketilfelle.client
+
+import io.ktor.client.call.*
+import io.ktor.client.features.*
+import io.ktor.client.request.*
+import io.ktor.client.statement.*
+import io.ktor.http.*
+import net.logstash.logback.argument.StructuredArguments
+import no.nav.syfo.client.httpClientDefault
+import no.nav.syfo.client.sts.StsClient
+import no.nav.syfo.domain.AktorId
+import no.nav.syfo.syfosyketilfelle.COUNT_CALL_SYFOSYKETILFELLE_OPPFOLGINGSTILFELLE_PERSON_FAIL
+import no.nav.syfo.syfosyketilfelle.COUNT_CALL_SYFOSYKETILFELLE_OPPFOLGINGSTILFELLE_PERSON_SUCCESS
+import no.nav.syfo.syfosyketilfelle.domain.KOppfolgingstilfellePerson
+import no.nav.syfo.util.bearerHeader
+import org.slf4j.LoggerFactory
+
+class SyfosyketilfelleClient(
+    baseUrl: String,
+    private val stsClient: StsClient,
+) {
+    private val httpClient = httpClientDefault()
+
+    private val syfosyketilfelleOppfolgingstilfellePersonUrl: String =
+        "$baseUrl$SYFOSYKETILFELLE_OPPFOLGINGSTILFELLE_PERSON_PATH"
+
+    suspend fun oppfolgingstilfellePerson(
+        callId: String,
+        aktorId: AktorId,
+    ): KOppfolgingstilfellePerson? {
+        try {
+            val stsToken = stsClient.token()
+            val url = "$syfosyketilfelleOppfolgingstilfellePersonUrl/${aktorId.value}"
+            val response: HttpResponse = httpClient.get(url) {
+                header(HttpHeaders.Authorization, bearerHeader(stsToken))
+                accept(ContentType.Application.Json)
+            }
+            COUNT_CALL_SYFOSYKETILFELLE_OPPFOLGINGSTILFELLE_PERSON_SUCCESS.increment()
+            if (response.status == HttpStatusCode.NoContent) {
+                return null
+            }
+            return response.receive<KOppfolgingstilfellePerson>()
```

Comment:
Jeg lurer på om dette vil føre til en `NoTransformationFoundException` hvis man får noe annet enn 200 med riktig objekt (og 204 som vi allerede har sjekket for). Fordi den prøver å gjøre om json ala `code: 401, message: "Unauthorized"` til et `KOppfolgingstilfellePerson`-objekt, og det funker ikke 🤔

Reply:
Med andre ord: Jeg tror det er for tidlig å telle "success"-metrikk på linje 38, og jeg tror ikke `catch`ene fungerer.
(Og en annen ting: Er det nødvendig med to ulike `catch`er? De gjør jo akkurat det samme?)

Reply:
1. Kan flytte metrikk et hakk ned. 
2. Jeg la til en test nå hvor en sjekker at catch fungerer. Den får 400 fra Syfosyketilfelle og så får den 400 ut videre, ser ut som om det fungerer fint 👍 
3. Tror ikke det går ann å catche flere exceptions i en one-liner. Ikke som jeg har klart å google meg frem til i hvert fall.